OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -143,6 +147,7 @@ public ObserverStatsLogic(World world, ModData modData, WorldRenderer worldRende
 				createStatsOption("Economy", ObserverStatsPanel.Economy, economyPlayerTemplate, () => DisplayStats(EconomyStats)),
 				createStatsOption("Production", ObserverStatsPanel.Production, productionPlayerTemplate, () => DisplayStats(ProductionStats)),
 				createStatsOption("Support Powers", ObserverStatsPanel.SupportPowers, supportPowersPlayerTemplate, () => DisplayStats(SupportPowerStats)),
+				createStatsOption("Army", ObserverStatsPanel.Army, armyPlayerTemplate, () => DisplayStats(ArmyStats)),
```

Comment:
The `statsDropDown.ShowDropDown` height should be made larger to avoid the scrollbar that this introduces.